vllm/entrypoints/openai/api_server.py

```diff
@@ -540,8 +551,8 @@ async def run_server(args, **uvicorn_kwargs) -> None:
     # workaround to make sure that we bind the port before the engine is set up.
     # This avoids race conditions with ray.
     # see https://github.com/vllm-project/vllm/issues/8204
-    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-    sock.bind(("", args.port))
+    sock_addr = ("", args.port)
+    sock = create_server_socket(sock_addr)
```

Comment:
This code changed in `main`, so you'll need to rebase the PR

Reply:
fixed